### DISCUSSION TITLE
Code Coverage in Trunk UI

### DISCUSSION THREAD

**Discussion by gewenyu99** [trunk-io/docs#278] [Feature Requests]
Originally posted by Anurag Kanungo

It would be amazing to see support to submit, visualize, track and gate on code coverage via turnk. Right now i still have to use a mix of similar tools where all local and PR linting/formatting is done by trunk, but code coverage is tracked/gated by other tools.

**Comment by Ryang20718** [comment]
+1 to this a year later
